### PR TITLE
Databrew Pipeline Example

### PR DESCRIPTION
**Pull request by b-akhil**
*Issue #, if available:* [**DatabrewTransformStage**](https://github.com/aws-samples/aws-ddk-examples/issues/18)

*Description of changes:* Added a Example Databrew Pipeline which leverages the DDK DataBrewFactory and DataBrewTransformStage.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by malachi-constant — COMMENTED** [review]
I was able to test a deployment and invocation and everything worked great! Thanks so much for the contribution.

File: databrew-athena/app.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+import aws_cdk as cdk
+
+from ddk_app.ddk_app_stack import DdkApplicationStack
+
+app = cdk.App()
+DdkApplicationStack(app, "DdkApplicationStack", "dev")
```
[the remark this answers is not in the document]
Comment (b-akhil) [reply databrew-athena/app.py:8]: Understood, Let me make the changes

[the remark this answers is not in the document]
Comment (b-akhil) [reply databrew-athena/app.py]: I have changed the stack name from `DdkApplicationStack` to  `DataBrewAthenaStack`. I also changed the file name.

Also made one another change with custom resource lambda handler for the deleting the recipe.

**Review by malachi-constant — APPROVED** [review]

File: databrew-athena/ddk_app/databrew_athena_stack.py
Comment (anmolsgandhi) [inline databrew-athena/ddk_app/databrew_athena_stack.py:1]: can you remove unused imports ?

**Review by anmolsgandhi — CHANGES_REQUESTED** [review]
Looks Good !! Left some minor comments

File: databrew-athena/ddk.json
[the remark this answers is not in the document]
Comment (b-akhil) [reply databrew-athena/ddk.json:1]: Removed the configs within ddk.json as they are not being used

File: databrew-athena/requirements.txt
[the remark this answers is not in the document]
Comment (b-akhil) [reply databrew-athena/requirements.txt:1]: Updated the DDK Version to 0.6.2

File: databrew-athena/ddk_app/databrew_athena_stack.py
Comment (b-akhil) [reply databrew-athena/ddk_app/databrew_athena_stack.py:1]: Removed the unused imports within databrew_athena_stack.py

**Review by anmolsgandhi — APPROVED** [review]

**Pull request merged**